snapshot_dbg_cli_tests/test_cli_version.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Unit test file for the cli_version module.
+"""
+
+import unittest
+import urllib
+
+import snapshot_dbg_cli.cli_version
+
+from email.message import EmailMessage
+from io import BytesIO
+from io import StringIO
+from unittest.mock import ANY
+from unittest.mock import patch
+from urllib.error import HTTPError
+from urllib.error import URLError
+
+
+class CliVersionTests(unittest.TestCase):
+  """ Contains the unit tests for the cli_version module.
+  """
+
+  def setUp(self):
+    self.urlopen_patcher = patch(
+        'snapshot_dbg_cli.http_service.urlopen', autospec=True)
+    self.urlopen_mock = self.urlopen_patcher.start()
+    self.addCleanup(self.urlopen_patcher.stop)
+
+    # Just to have a default return value set, tests that care will update it.
+    self.set_urlopen_response('foo')
+
+  def set_urlopen_response(self, body):
+    response_headers = EmailMessage()
```

Comment:
Why EmailMessage?

Reply:
Yes, that does look odd, but that's what is used. I've added in a comment and link to  https://docs.python.org/3/library/urllib.request.html#urllib.response.addinfourl.headers